README.md

```diff
@@ -41,87 +41,94 @@
 </h3>
 <br/>
 
+# Overview
```

Comment:
I think that we need a more straightforward introduction to what `astro` is. I took my best shot at creating an Overview section here, but please add more / less detail depending on how you view this project. I also thought including a before/ after section would make it even more clear what value `astro` adds to your Airflow project.